OpenRA.Mods.Common/Widgets/Logic/Installation/DownloadPackagesLogic.cs

```diff
@@ -52,9 +57,9 @@ void ShowDownloadDialog()
 
 			var cancelButton = panel.Get<ButtonWidget>("CANCEL_BUTTON");
 
-			var mirrorsFile = Platform.ResolvePath("^", "Content", Game.ModData.Manifest.Mod.Id, "mirrors.txt");
+			var mirrorsFile = Platform.ResolvePath("^", "Content", "mirrors.txt");
```

Comment:
It can have a `Path.GetRandomFileName()` too (and don't forget to remove it when you're done).